server/Helpers/signIn.js

```diff
@@ -3,6 +3,13 @@ import jwt from "jsonwebtoken";
 // todo: reduce this time and implement refresh token
 const ACCES_TOKEN_EXPIRATION = 86400; // 24h in secondes
 
+// const accestTokenName = "accesToken";
+const accesTokenOption = {
+  httpOnly: true
+  // todo: add `secure: true` to only send token in https
+  // secure: true,
+};
```

Comment:
Nice but: the goal of this was to make easy the change of the `accesToken` option by using the **same variable** in `createAccestToken ` AND `deleteAccessTokenCookie `.

With what you made it's more difficult that before. If we want to add an option we need to add it in **2 variable** in **2 different file**

You have 2 solution:
- Put back `createAccestToken ` and `deleteAccessTokenCookie ` in the same file and use the **same variable** 
- export the variable (`accestTokenName` and `accesTokenOption`) in `server/Helpers/signIn.js` and import them in `server/Helpers/signOut.js`